docs/maintain-guides-society-kusama.md

```diff
@@ -0,0 +1,168 @@
+
+You can find an existing member to place a bid on your behalf if you do not have KSM and you are willing to give them a tip. An existing member can submit a `vouch(who,value,tip)` transaction.
+
+> vouch(who,value,tip)
+>
+> who - Which user you are vouching for
+>
+> value - The value that the user would like to get when joining the society
+> 
+> tip - Fees you get
+>
+> Note: The final value that the candidate will get = (value - tip)
+
+![Society Dashboard](assets/society/vouch.jpg)
+
+
+### 2. Candidate Phase
+
+![Society Vote Candidate](assets/society/vote_candidate.jpg)
+
+> vote(candidate,approve)
+>
+> candidate - Select the candidate you would like to vote
+>
+> approve - Yes / No
+
+Bids selected in this phase will be voted on by the existing members to decide whether or not you will be approved to join the society. Members will vote for all the candidates and the final outcome will be randomly selected by one of the votes. Let's take a look the example shown as below:
+
+> **_NOTE:_** 
+>
+> If the randomly selected member does not vote, it will be treated as a rejection.
+> 
+> For each rotation period, the maximum number of members that can be accepted is set as 10. 
+
+A - Accept,  R - Reject,  S - Skeptic
+
+Member   | 1 | 2 | 3 | 4 | 5 |
+-------- | - | - | - | - | - |
+Vote     | A | A | A | R | S |
+Selected |   |   | X |   |   | 
+
+In this example, a candidate will be approved to join the society since member 3 was selected as a final voting outcome. A number of members will also be randomly chosen as "skeptics" to vote for the candidates during the rotation period. Since member 5 was chosen as a skeptic, they are required to participate in the voting process. If they do not participate in voting, they will be punished with one strike per missing vote. If one accumulates too many strikes, one's membership is suspended which means they may need to re-apply and their unclaimed payouts will be slashed. Moreover, each member who voted opposite to the randomly selected vote will have their unclaimed payouts slashed and strikes increased. In this case, member 4 will be punished.
+
+> **_NOTE:_** 
+>
+> The maximum number of strikes you can have is 10 in Kusama.
+
+These slashed funds (It is set as `2` KSM currently) will be given to a random member who voted the same as the selected vote as a reward for participating in the vote. It is also required to lock-up for some period of time before the member can get it.
+
+#### Lock-up Time
+
+It would take the number of members of the society as the variable to determine how many blocks you have to wait in order to get the payout. The longest lock-up time is closer to 3 years. The formula is available [here](https://github.com/paritytech/substrate/blob/master/frame/society/src/lib.rs#L1588) if you would like to have a look.
```

Comment:
We should not link to lines of code. Not even to source files, really, but better than lines of code.

```suggestion
It would take the number of members of the society as the variable to determine how many blocks you have to wait in order to get the payout. The longest lock-up time is closer to 3 years. The formula is defined [in the society pallet](https://github.com/paritytech/substrate/blob/master/frame/society/) if you would like to have a look.
```